WorkflowManagement/WorkflowManagement.csproj

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you use the new csproj file type for this? That way we can retarget to .net core/.net framework much easier.